html/browsers/history/the-location-interface/location-ancestororigins.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<iframe src=/common/blank.html></iframe>
+<iframe src=/common/blank.html referrerpolicy=no-referrer></iframe>
+<script>
+test(() => {
+  assert_equals(location.ancestorOrigins.length, 0)
+}, "location.ancestorOrigins basic check")
+
+async_test(t => {
+  const frame = document.querySelector("iframe")
+  frame.onload = t.step_func_done(() => {
```

Comment:
This is racy.  It might have already finished loading, and then your test will time out.  You need to set the src _after_ you set up the load listener.